opentech/users/urls.py

```diff
@@ -0,0 +1,15 @@
+from django.conf.urls import url
+
+from django.contrib.auth import views as auth_views
+
+urlpatterns = [
+    url(r'^$', auth_views.login, {
+        'template_name': 'users/login.html'
+    }, name='login'),
```

Comment:
You cant have two urls with the same name. any attempt to reverse will error·

Reply:
Also I'm not sure we need the first url at the moment, just the `/login` is enough.

Reply:
This is /user and /user/login, both requiring login. Will replace the first one with the dashboard view with the `login_required` when we start building the dashboard